### PR TITLE
Java example for 0.15.2

### PR DESCRIPTION
**Pull request by radetsky**
New Java project, but old source code. I created a new Java project using `gradle init` and moved the source code from the previous example. Added simple README to describe how to run the example.

- [x] Example projects and code samples are up-to-date (in case of API changes)

### CODE REVIEW

**Review by vixentael — APPROVED** [review]

**Comment by vixentael** [comment]
uwuwuw still failing :(

**Review by vixentael — COMMENTED** [review]
let's see if these changes fix the tests!

File: .github/workflows/test-java.yaml [hunk 1]
```diff
@@ -112,13 +112,13 @@ jobs:
...
     name: Example project – JavaThemis
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-22.04
     steps:
       - name: Install system dependencies
         run: |
           sudo sh -c 'echo "DEBIAN_FRONTEND=noninteractive" >> /etc/environment'
           sudo apt update
-          sudo apt install --yes gcc make libssl-dev default-jdk
+          sudo apt install --yes gcc make libssl-dev default-jdk dpkg-dev
```
Comment (Lagovas) [inline .github/workflows/test-java.yaml:121]: for what we need dpkg-dev?

Comment (radetsky) [reply .github/workflows/test-java.yaml:121]: For dpkg-architecture. I tested the solution on a real OS installed on Digital Ocean.

**Pull request merged**